data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg

```diff
@@ -44,15 +44,15 @@
 
         {STORY_PART_TALLIN (_ "What do you mean?")}
 
-        {STORY_PART_HAMEL (_ "Yer gold, Tallin, we ha’ failed to protect it.")}
+        {STORY_PART_HAMEL (_ "Yer gold, Tallin, we failed to protect it.")}
 
         {STORY_PART_TALLIN (_ "My gold!? Why, what happened? Start from the beginning.")}
 
         {STORY_PART_HAMEL (_ "No sooner had ye left then the orcs once again launched a massive assault.")}
 
-        {STORY_PART_HAMEL (_ "Such a massive and fierce assault it was that we had to call upon all our reserves. When that proved insufficient, we had nae other choice then to call some men off the other defenses.")}
+        {STORY_PART_HAMEL (_ "Such a massive and fierce assault it was that we had to call upon all our reserves. When that proved insufficient, we had nae other choice then to call some dwarves off the other defenses.")}
```

Comment:
We've had the conversation before about using 'men' to refer to non-humans, eg #4880, but I suppose this change is okay.

Reply:
Um, OK?